services/security/authentication/casigned/casigned.go

```diff
@@ -37,6 +44,47 @@ func (c *client) Login(ctx context.Context, group string, identity *security.Ide
 	return &response.Token, nil
 }
 
+func RenewRoutine(group, server string) {
+	renewalAttempt := 0
+	// Waiting for a few seconds to avoid spamming short-lived sdk user
+	time.Sleep(time.Second * 5)
+	for {
+		wssdConfig := auth.WssdConfig{}
+		err := marshal.FromJSONFile(auth.GetWssdConfigLocation(), &wssdConfig)
+		if err != nil {
+			fmt.Printf("Failed to open config file in location %s: %v\n", auth.GetWssdConfigLocation(), err)
+			panic("Failed to open config file")
```

Comment:
Can we either return an error, or log the error and return gracefully instead of panicking? Keep in mind that panic will crash the app that uses the SDK. This typically is used by attackers to generate denial of service attacks. So unless the state of the program is unrecoverable (i.e. bad memory corruption), we should try to be graceful.

Here we seem to be in a callback, so I understand we don't have the ability to return the error to the caller, but we should log the error and just return to the caller. If we fail to renew CA certs, the user will notice it very quickly.